test/jsonpointer/jsonpointer_error_test.cc

```diff
@@ -1,6 +1,7 @@
 #include <gtest/gtest.h>
 #include <sourcemeta/core/json.h>
 #include <sourcemeta/core/jsonpointer.h>
+#include <sourcemeta/core/jsonpointer_error.h>
```

Comment:
This makes the includes better but should the consumer of `jsonpointer` be aware of more than one header file?

Reply:
I think it's fine for test files!